app/views/historic_appointments/show.html.erb

```diff
@@ -3,10 +3,13 @@
 
 <header class="block headings-block">
   <div class="inner-block floated-children">
-    <%= render partial: 'shared/heading',
-              locals: { type: link_to('History', "/government/history", class: "govuk-link"),
-                        heading: "Past #{@role.name.pluralize}",
-                        big: true } %>
+    <%= render "govuk_publishing_components/components/title", {
+      context: {
+        text: "History",
+        href: "/government/history",
+      },
+      title: "Past #{@role.name.pluralize}",
+    } %>
```

Comment:
Would need to check this on intergration.

Reply:
I don't mind the extra space for this template. These pages in particular need a lot of work anyway which I believe is planned in our backlog. How do you feel about me leaving this alone for now and "fixing" it when we get to this template in our redesign work?